.pubnub.yml

```diff
@@ -410,23 +410,26 @@ features:
 supported-platforms:
   - version: PubNub Java SDK
     platforms:
-      - Windows 10 (7u85 and above)
+      - Windows 10 (8u51 and above)
       - Windows 8.x (Desktop)
       - Windows 7 SP1
       - Windows Vista SP2
-      - Windows Server 2008 SP2 and 2008 R2 SP1 (64-bit)
-      - Windows Server 2012 (64-bit) and 2012 R2 (64-bit)
-      - Intel-based Mac running Mac OS X 10.7.3 (Lion) or later.
+      - Windows Server 2008 R2 SP1 (64-bit)
+      - Windows Server 2012 and 2012 R2 (64-bit)
+      - Intel-based Mac running Mac OS X 10.8.3+, 10.9+
```

Comment:
I wonder why they call out "10.9+" specifically, since it's implied in the "10.8.3+"?

Reply:
Don't care...